lib/ace/ext/papeeria/katex-previewer.coffee

```diff
@@ -109,12 +104,9 @@ define((require, exports, module) ->
 
     updateRange: ->
       { row: cursorRow, column: cursorColumn } = @editor.getCursorPosition()
-      {
-        outer: @curOuterRange
-        inner: @curInnerRange
-        start: @curStartId
-        end: @curEndId
-      } = @equationRangeHandler.getEquationRange(cursorRow, cursorColumn)
+      @currentRange = @equationRangeHandler.getEquationRange(cursorRow, cursorColumn)
+      if not @currentRange
+        @ok = false
```

Comment:
I can't say that I undertand the meaning of ok. Here it is set to `false` when `currentRange`  is null. Okay. However, at line 163 `currentRange` is set to null and ok is set to true. I may think that ok means "ok to show preview" but `updateRange` is called when preview is already shown, no? What's the purpose of changing ok here? 

I think `ok` worth at least renaming,so that it was clear what doesit mean, and maybe removed completely. It appears that in the couple of places (lines 132, 151) it can be replaced with boolean return value from `updateRange`, lines 180-183 and line 164 seem to be duplicating themselves with respect to setting ok to true (is it possible that `currentContext != equation, not ok and not contextPreviewExists` at the same time?)